src/store/models/taro.ts

```diff
@@ -1,16 +1,33 @@
 import { action, Action, thunk, Thunk, thunkOn, ThunkOn } from 'easy-peasy';
-import { Status, TaroNode } from 'shared/types';
+import { LightningNode, Status, TarodNode, TaroNode } from 'shared/types';
 import * as PTARO from 'lib/taro/types';
 import { StoreInjections } from 'types';
 import { RootModel } from './';
 
+import * as TARO from 'shared/tarodTypes';
+//import { useStoreActions, useStoreState } from 'store';
+//import React, { useState } from 'react';
+import { BLOCKS_TIL_CONFIRMED, TARO_ASSET_TYPE } from 'utils/constants';
+
 export interface TaroNodeMapping {
   [key: string]: TaroNodeModel;
 }
 
 export interface TaroNodeModel {
   assets?: PTARO.TaroAsset[];
   balances?: PTARO.TaroBalance[];
+  batchKey?: string;
+}
+
+export interface MintAssetPayload {
+  node: TaroNode;
+  assetType: boolean;
```

Comment:
This should be a `string` containing either `NORMAL` or `COLLECTIBLE`, not a boolean.

Reply:
Done